runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -108,12 +111,18 @@ public void changedUpdate(DocumentEvent e)
 	final void rebuildPluginList()
 	{
 		Map<String, JPanel> newChildren = new TreeMap<>();
-		configManager.getConfigProxies().stream()
-			.map(configManager::getConfigDescriptor)
-			.filter(configDescriptor -> configDescriptor.getItems().stream()
-				.anyMatch(cid -> !cid.getItem().hidden()))
-			.forEach(cd ->
+		configManager.getConfigProxies()
+			.stream()
+			// Config cannot be key because Proxy does not implement hashCode
+			.collect(Collectors.toMap(configManager::getConfigDescriptor, Function.identity()))
```

Comment:
But it is ConfigDescriptor, not proxy to anything if I am not wrong, also, I think in stream, noone cares about hashCode or equals, because it is just stream. Even if that was false, cleaner solution would be then to add equals and hashCode to ConfigDescriptor.

Reply:
Config has to be passed to openGroupConfigPanel in some way, shape or form.
Function.identity() is Config, a Proxy, and can thus not be the key.

Reply:
Ahh now I understand. Still, do not really like this solution, maybe just map it to Stream<Map.Entry> directly instead of collecting it to map first and then iterating through entryset etc ect, you can freely construct Map.Entry I believe.